app/src/main/java/com/github/se/signify/ui/common/Screens.kt

```diff
@@ -242,3 +247,35 @@ fun BottomBar(navigationActions: NavigationActions) {
       tabList = LIST_TOP_LEVEL_DESTINATION,
       selectedItem = navigationActions.currentRoute())
 }
+
+@Composable
```

Comment:
You copied it from the navigation folder, it makes more sense here since it's only used here.